### PR TITLE
feat: added timeplus input

### PR DESCRIPTION
**Pull request by ye11ow**

### CODE REVIEW

File: internal/impl/timeplus/input.go [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+	"github.com/redpanda-data/benthos/v4/public/service"
+	"github.com/redpanda-data/connect/v4/internal/impl/timeplus/driver"
+	"github.com/redpanda-data/connect/v4/internal/impl/timeplus/http"
+)
+
+var inputConfigSpec *service.ConfigSpec
+
+func init() {
+	inputConfigSpec = service.NewConfigSpec().
```
Comment (ye11ow) [inline internal/impl/timeplus/input.go:19]: Please help review the docs here.

File: docs/modules/components/pages/inputs/timeplus.adoc [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  timeplus:
+    query: select * from iot # No default (required)
+    url: tcp://localhost:8463
+    workspace: "" # No default (optional)
+    apikey: "" # No default (optional)
+    username: "" # No default (optional)
+    password: "" # No default (optional)
+```
+
+This input can execute a query on Timeplus Enterprise Cloud, Timeplus Enterprise (self-hosted) or Timeplusd. A structured message will be created
```
Comment (jovezhong) [inline docs/modules/components/pages/inputs/timeplus.adoc:41]: Maybe changing the 3rd option from `Timeplusd` to `Timeplus Proton`. It's a bit confusing what's the difference in this context for 2nd and 3rd option.

File: docs/modules/components/pages/inputs/timeplus.adoc [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+from each row received.
+
+If it is a streaming query, this input will keep running until the query is terminated. If it is a table query, this input will shuts down once the rows from the query are exhausted.
```
Comment (jovezhong) [inline docs/modules/components/pages/inputs/timeplus.adoc]: "will shuts down" to "will shut down"

File: docs/modules/components/pages/inputs/timeplus.adoc [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+output:
+  timeplus:
+    url: http://localhost:8000
+    workspace: my_workspace_id
+    query: select * from iot
+    username: username
+    password: pw```
+
+--
+From Timeplusd::
```
Comment (jovezhong) [inline docs/modules/components/pages/inputs/timeplus.adoc]: For Timeplus Core Engine (either `timeplusd` in Timeplus Enterprise, or Timeplus Proton)

File: internal/impl/timeplus/input.go (hunk 1 above)
Comment (jovezhong) [reply internal/impl/timeplus/input.go:19]: I gave the feedback in the addoc. Maybe it's generated via the following code. Sorry, pls refer to earlier comments

**Review by jovezhong — COMMENTED** [review]
shared some comments for the doc

**Pull request closed without merging**